documentation/Intrinsics.md

```diff
@@ -683,3 +685,103 @@ CACHESIZE, EOF, FP_CLASS, INT_PTR_KIND, ISNAN, LOC
 MALLOC
 ```
 
+# Intrinsic Procedure Support in f18
+This section gives an overview of the support inside f18 libraries for the
+intrinsic procedures listed above.
+It may be outdated, refer to f18 code base for the actual support status.
+
+## Semantic Analysis
+F18 semantic expression analysis phase detects intrinsic procedure references,
+validates the argument types and deduces the return types.
+This phase currently supports all the intrinsic procedures listed above but the ones in the table below.
```

Comment:
Could you share the plan for the ones which are not supported? Are these work in progress?

Reply:
Basically the same plan as for MOD folding, the remainder of the work will be done later when we are closer to executing code.